app/plugin.go

```diff
@@ -1121,8 +1121,8 @@ func (ch *Channels) getPluginStateOverride(pluginID string) (bool, bool) {
 			return true, false
 		}
 	case model.PluginIdCalls:
-		if model.IsCloud() {
-			return true, ch.cfgSvc.Config().FeatureFlags.CallsEnabled
+		if !ch.cfgSvc.Config().FeatureFlags.CallsEnabled {
```

Comment:
Given this feature flag existed before, and presumably defaulted to false, will existing installations that chose to enable it discover it disabled via this feature flag unexpectedly?

Reply:
Merging but please let's continue the discussion here if necessary.

Reply:
I think I was worried that `CallsEnabled: false` might have been written to `config.json` in a past version, and now it would "force it off," but looking at my own local installation, it seems we don't persist these to the actual config anyway, so it sounds like a non-issue. Thanks for the follow-up!